### ISSUE TITLE
Easier Certificate Error Workaround

### ISSUE THREAD

**Issue by paragonie-scott** [microsoftgraph/php-connect-sample#17]
From the README:

> ### Error: Unable to get local issuer certificate
> 
> You receive the following error after providing your credentials to the sign in page.
> ```
> SSL certificate problem: unable to get local issuer certificate
> ```
> 
> cURL can't verify the validity of the Microsoft certificate when trying to issue a request call to get tokens. You must configure cURL to use a certificate when issuing https requests by following these steps:  
> 
> 1. Download the cacert.pem file from [cURL website](http://curl.haxx.se/docs/caextract.html). 
> 2. Open your php.ini file and add the following line
> 
> 	```
> 	curl.cainfo = "path_to_cacert/cacert.pem"
> 	```
> 

There's actually a much simpler way to solve this problem, which is worth using as an alternative.

1. Add `paragonie/certainty` as a dependency.
2. Use [RemoteFetch](https://github.com/paragonie/certainty/blob/master/docs/features/RemoteFetch.md#remotefetch) to ensure the latest CACert bundle is available.

Certainty ensures that the downloaded CACert.pem file is cryptographically signed by our security team using Ed25519 and that the metadata is checked into an append-only cryptographic ledger. The sha256sum of each bundle is also published, so that it can be cross-correlated with the bundle shared by the cURL developers. We have a [detailed write-up](https://paragonie.com/blog/2017/10/certainty-automated-cacert-pem-management-for-php-software) about its motivation and design, if you're interested.

In short: Given access to the Internet and a writable directory, it guarantees that the PHP program in question has the latest CACert.pem bundle installed and delivers them [securely](https://defuse.ca/triangle-of-secure-code-delivery.htm).

**Comment by jasonjoh** [comment]
@paragonie-scott we are archiving this sample as it's no longer being maintained. We're replacing it with https://github.com/microsoftgraph/msgraph-training-phpapp.
